manager/src/main/java/com/redhat/service/bridge/manager/api/user/ErrorsAPI.java

```diff
@@ -45,11 +54,25 @@
     @Inject
     BridgeErrorService service;
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = ErrorListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401")
+    })
+    @Operation(summary = "Get the list of errors.", description = "Get the list of errors from the error catalog.")
     @GET
     public Response getErrors(@Valid @BeanParam QueryInfo queryInfo) {
         return Response.ok(ListResponse.fill(service.getUserErrors(queryInfo), new ErrorListResponse(), ErrorResponse::from)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeError.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401")
```

Comment:
Just unrelated note, putting non existing id results in NPE:
```
Caused by: java.lang.NullPointerException
	at com.redhat.service.bridge.infra.exceptions.BridgeErrorInMemoryDAO.findErrorByIdAndType(BridgeErrorInMemoryDAO.java:110)
```

Reply:
Should return 404 I guess. I will create a ticket for that